CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Tracking/AnalyticsProperties.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  AnalyticsProperties.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+
+import Foundation
+struct AnalyticsProperty {
+    let key: AnalyticsPropertyKey
+    var value: AnalyticsPropertyValue
+}
+
+protocol AnalyticsPropertyValue {
+    func analyticsPropertyValue() -> Self
+}
+
+extension String: AnalyticsPropertyValue {
+    func analyticsPropertyValue() -> String {
+        return self
+    }
+}
+
+extension Int: AnalyticsPropertyValue {
+    func analyticsPropertyValue() -> Int {
+        return self
+    }
+}
+
+extension UInt: AnalyticsPropertyValue {
```

Comment:
[Clarification]: Do we plan to pass the properties `UInt` and `Double` if we are not sure - I propose to clean it and add it later when we need it. 😇

Reply:
I cleaned them in another pr 🙈 .